src/components/Indicators/LoadingIndicator.js

```diff
@@ -1,65 +1,55 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import styled from '@stream-io/styled-components';
 import PropTypes from 'prop-types';
+
+import { TranslationContext } from '../../context';
 import { Spinner } from '../Spinner';
-import { withTranslationContext } from '../../context';
 
 const Container = styled.View`
+  align-items: center;
   height: 100%;
   justify-content: center;
-  align-items: center;
   ${({ theme }) => theme.loadingIndicator.container.css}
 `;
 const LoadingText = styled.Text`
-  margin-top: 20px;
   font-size: 14px;
   font-weight: 600;
+  margin-top: 20px;
   ${({ theme }) => theme.loadingIndicator.loadingText.css}
 `;
 
-class LoadingIndicator extends React.PureComponent {
-  static propTypes = {
-    listType: PropTypes.oneOf(['channel', 'message', 'default']),
-    loadingText: PropTypes.string,
-  };
+const LoadingIndicator = (props) => {
+  const { listType, loadingText } = props;
```

Comment:
are we using a spread of props somewhere else or should we just destructure this in the method param?